app/src/playStore/java/com/eventyay/organizer/core/event/create/GooglePlacePicker.java

```diff
@@ -0,0 +1,67 @@
+package com.eventyay.organizer.core.event.create;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Intent;
+
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.GoogleApiAvailability;
+import com.google.android.gms.common.GooglePlayServicesNotAvailableException;
+import com.google.android.gms.common.GooglePlayServicesRepairableException;
+import com.google.android.gms.location.places.Place;
+import com.google.android.gms.location.places.ui.PlacePicker;
+
+import timber.log.Timber;
+
+public class GooglePlacePicker {
+
+    private final GoogleApiAvailability googleApiAvailabilityInstance = GoogleApiAvailability.getInstance();
+    private static final int PLACE_PICKER_REQUEST = 1;
+    private Place place;
+    private boolean showLocation = false;
+
+    public void onSelectingPlace(Activity activity) {
```

Comment:
This is still misleading, rename to `launchPicker` and make it return a boolean which denotes `!showLocation`

So there'll be no need of storing `showLocation` variable and two method calls in the calling class will reduce to one